litellm/integrations/langfuse.py

```diff
@@ -437,6 +437,7 @@ def _log_langfuse_v2(
             generation_params = {
                 "name": generation_name,
                 "id": clean_metadata.pop("generation_id", generation_id),
+                "parent_observation_id": metadata.get("parent_observation_id"),
```

Comment:
this would raise an error if the value is None i believe, as the dictionary would now contain a 'None' object

can you instead have this below in an if metadata.get('parent_observation_id') is not None statement

Reply:
I have this PR branch in production in our application it does not raise an error. 
But I will add it anyway to make sure, that future Langfuse updates do not potentially break this!